packages/core/src/components/portal/portal.tsx

```diff
@@ -10,10 +10,9 @@ import * as ReactDOM from "react-dom";
 
 import * as Classes from "../../common/classes";
 import * as Errors from "../../common/errors";
-import { IProps, removeNonHTMLProps } from "../../common/props";
-import { safeInvoke } from "../../common/utils";
+import { IProps } from "../../common/props";
 
-export interface IPortalProps extends IProps, React.HTMLProps<HTMLDivElement> {
+export interface IPortalProps extends IProps, React.HTMLAttributes<HTMLDivElement> {
```

Comment:
Remind me what this change is about? I remember seeing something before my trip, but I've forgotten.